flights/berks_ceh_homeless_enrollments_flight.yaml

```diff
@@ -162,15 +188,16 @@ entityDefinitions:
         column: "TimesHomelessPastThreeYears"
         transforms:
         - !<transforms.ReplaceTransform>
-          target: ["1", "2", "3", "4"]
-          goal: ["1", "2", "2", "4"]
+          target: ["1", "2", "3", "4", "8", "9", "99"]
+          goal: ["1", "2", "3", "4", "", "", ""]
+        - !<transforms.ParseIntTransform> {}
```

Comment:
here's where you actually need a parse int. The property is an integer. It goes linearly, so first you replace with the numbers you want, then you call the parseInt transformation.